cmd/cluster/core/create.go

```diff
@@ -97,6 +97,7 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.Arch, "arch", opts.Arch, "The default processor architecture for the NodePool (e.g. arm64, amd64)")
 	flags.StringVar(&opts.PausedUntil, "pausedUntil", opts.PausedUntil, "If a date is provided in RFC3339 format, HostedCluster creation is paused until that date. If the boolean true is provided, HostedCluster creation is paused until the field is removed.")
 	flags.StringVar(&opts.ReleaseStream, "release-stream", opts.ReleaseStream, "The OCP release stream for the cluster (e.g. 4-stable-multi), this flag is ignored if release-image is set")
+	flags.StringVar(&opts.FailureDomain, "failure-domain", opts.FailureDomain, "The failure domain for the NodePool")
```

Comment:
This is adding it to `create cluster`, right? Why would it be required here? I'd have thought it was only required for `create nodepool`.

Reply:
yeah I know, I'll have to look again but this is how other nodepool options were added too.